controller/controller.js

```diff
@@ -11,6 +11,18 @@ const getTopics = (req,res,next) => {
     
 }
 
+const getCommentsByArticleId = (req, res, next) => {
+  const { article_id } = req.params;
```

Comment:
You will need to check the reason for the query returning empty rows:
- article exists but has no comments.
- article does not exist.
You will need to check whether the article exists in order to determine this.
I recommend the use of Promise.all() for this to make the code more efficient.